src/middlewares/validators/schemas/order-schemas.js

```diff
@@ -25,8 +25,15 @@ const createOrder = Joi.object({
   }),
 });
 
+const orderIdSchema = Joi.object({
```

Comment:
change this schema to validate body values as well

order delivery status can be changed only to specific values, you should validate that on request validation step

otherwise mongoose will throw an error